index.js

```diff
@@ -1,13 +1,44 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
-
-
 /***** Deliverable 2 *****/
 
+const header = document.querySelector('h1').style.color = 'red'
+//console.log("Here's your header:", header)
+
 
 /***** Deliverable 3 *****/
-// console.log("PLAYERS array looks like this:", PLAYERS)
+const list = document.createElement('ul')
+const div = document.querySelector('.player-container')
+
+div.append(list)
+
+function renderPlayer(playerObj){
+
+    const player = document.createElement("li")
+    player.className = "playerObj"
+
+    player.innerHTML = `
+    <div class="player" data-number= "${playerObj.number}">
+        <h3>
+            ${playerObj.name} (<em>${playerObj.nickname}</em>)
+        </h3>
+        <img src="${playerObj.photo}" alt="${playerObj.name}">
+    </div>
+
+    `
+ //console.log("PLAYERS array looks like this:", PLAYERS)
+
+    list.append(player)
+}
```

Comment:
For this deliverable, you're adding more elements to the DOM than is necessary. We want to end up with HTML that looks like this:

```html
<div class="player-container">
  <div class="player" data-number="11">
    <h3>
      Mo Salah (<em>The Egyptian King</em>)
    </h3>
    <img src="https://cdn.cnn.com/cnnnext/dam/assets/190501145802-mo-salah-exlarge-169.jpg" alt="Mo Salah">
  </div>
  <div class="player" data-number="9">
    <h3>
      Roberto Firmino (<em>Bobby Chompers</em>)
    </h3>
    <img src="https://vignette.wikia.nocookie.net/liverpoolfc/images/f/f3/RFirmino2019.jpeg/revision/latest/scale-to-width-down/340?cb=20190807042207" alt="Roberto Firmino">
  </div>
...
```

With your code, we'll end up with this instead:

```html
<div class="player-container">
  <ul>
    <li class="playerObj">
      <div class="player" data-number="11">
        <h3>
          Mo Salah (<em>The Egyptian King</em>)
        </h3>
        <img src="https://cdn.cnn.com/cnnnext/dam/assets/190501145802-mo-salah-exlarge-169.jpg" alt="Mo Salah">
      </div>
    </li>
    <li class="playerObj">
      <div class="player" data-number="9">
        <h3>
          Roberto Firmino (<em>Bobby Chompers</em>)
        </h3>
        <img src="https://vignette.wikia.nocookie.net/liverpoolfc/images/f/f3/RFirmino2019.jpeg/revision/latest/scale-to-width-down/340?cb=20190807042207" alt="Roberto Firmino">
      </div>
    </li>
...
```

In this case, the `<ul>` element from line 9 isn't necessary, and the `<li>` elements from line 16 aren't necessary.  Have a look at the [lecture code](https://github.com/learn-co-students/nyc-web-091420/blob/master/21-js-mini-challenge-dom-review/index.js) to see how to solve this without creating extra DOM elements!